### PR TITLE
README Cleanup

### LINKED ISSUE #3359: Request) Clean up README.md!
The related modules section is probably not needed anymore bootstrap 4 offers a lot of these components and utilities, I don't think you need to add any extra features. It also links to version 3 rather than 4.

### PR DESCRIPTION
**Pull request by bpas247**
Fixes #3359

I'm not entirely sure if we should incorporate this change yet, as I feel like at least one of these libraries should be kept (or more related libraries should be added). 

Further discussion is needed before merge.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,12 +16,6 @@ To find the documentation for the latest Bootstrap 3 compatible release, check o
 
 If you would like to update React-Bootstrap within an existing project to use Bootstrap 4, please read our docs for **[migrating to React-Bootstrap V1](https://react-bootstrap.netlify.com/migrating/)**.
 
-## Related modules
-
-* [react-router-bootstrap][react-router-bootstrap] – Integration with [React Router][react-router]
-* [React Bootstrap Extended][react-bootstrap-extended] - A version of React Bootstrap where each component has extra props that map to bootstrap's utility classes. For example, `pullRight` adds the class `pull-right` to a component.
-* [Awesome React Bootstrap Components][awesome-react-bootstrap-components] - Additional components like off-canvas navbar, switch and sliders.
```
Comment (jquense) [inline README.md]: this is probably still relevant no?

File: README.md [hunk 2]
```diff
@@ -16,12 +16,6 @@ To find the documentation for the latest Bootstrap 3 compatible release, check o
 
 If you would like to update React-Bootstrap within an existing project to use Bootstrap 4, please read our docs for **[migrating to React-Bootstrap V1](https://react-bootstrap.netlify.com/migrating/)**.
 
-## Related modules
-
-* [react-router-bootstrap][react-router-bootstrap] – Integration with [React Router][react-router]
```
Comment (jquense) [inline README.md]: lets keep this, it's pretty agnostic to the library version

File: README.md (hunk 1 above)
Comment (bpas247) [reply README.md]: it's still being maintained (last commit 8 days ago), so it seems relevant to this repository. I think we should still include it

**Comment by guledali** [comment]
@bpas247 Why not remove this one too?
* [Awesome React Bootstrap Components][awesome-react-bootstrap-components] - Additional components like off-canvas navbar, switch and sliders.

The offcanvas has already been applied in bootstrap 4 https://getbootstrap.com/docs/4.1/examples/offcanvas/

Same goes for the slider https://getbootstrap.com/docs/4.1/components/forms/#range

Next release the will add the switch https://github.com/twbs/bootstrap/issues/22090

**Comment by guledali** [comment]
@bpas247 Why not remove this one too?
* [Awesome React Bootstrap Components][awesome-react-bootstrap-components] - Additional components like off-canvas navbar, switch and sliders.

The offcanvas has already been applied in bootstrap 4 https://getbootstrap.com/docs/4.1/examples/offcanvas/

Same goes for the slider https://getbootstrap.com/docs/4.1/components/forms/#range

Next release v4.2 the will mostly like add the switch aswell

**Comment by bpas247** [comment]
@guledali offcanvas, slider, and switch are not yet currently supported in the newest version of react-bootstrap (as of v1.0.0-beta.3), so it still seems relevant.

**Pull request merged**